core-bundle/tests/Routing/ResponseContext/CoreResponseContextFactoryTest.php

```diff
@@ -65,9 +64,7 @@ public function testResponseContext(): void
             $this->createMock(UrlGeneratorInterface::class),
         );
 
-        $responseContext = $factory->createResponseContext();
-
-        $this->assertInstanceOf(ResponseHeaderBag::class, $responseContext->getHeaderBag());
```

Comment:
This does not check anything anymore?

Reply:
See https://github.com/contao/contao/pull/7711#discussion_r1837898458